### PR TITLE
completed testing functions with jest and enzyme

### PR DESCRIPTION
**Pull request by DeonFranklin**

### CODE REVIEW

File: src/components/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  it("displays a navlink on the header", () => {
+    //Arrange the Header
+
+    //Act on the header by finding an element
+    const navLinkWrapper = header.find("NavLink");
+    expect(navLinkWrapper.length).toEqual(3);
+  });
+  
+  
+  it("checks for a tag by its class name to contain some text", () => {
```
Comment (WalkerTexas-coder) [inline src/components/Header.test.js]: This "it" statement is a bit off for what this test is doing. It's alright to leave this check mark undone if you don't have a class to call upon.

File: src/pages/Home.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
 import React, { Component } from 'react'
+import cuteasscat from '../assets/cuteasscat.png'
 
 class Home extends Component{
   render(){
     return(
-      <h1>Welcome to cat tinder!</h1>
+      <>
+      <h1><strong>Welcome to Cat Tinder!
```
Comment (WalkerTexas-coder) [inline src/pages/Home.js]: Please clean up the indentation on this page.

File: src/pages/Home.test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import Home from './Home'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe("When Home Loads...",() => {
+  it("displays a home cat image", () => {
```
Comment (WalkerTexas-coder) [inline src/pages/Home.test.js:9]: Excellent "it" statement

File: src/pages/NotFound.test.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import NotFound from './NotFound'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe("When NotFound Loads...",() => {
+  it("displays a not dound cat image", () => {
```
Comment (WalkerTexas-coder) [inline src/pages/NotFound.test.js]: Spelling error

**Review by WalkerTexas-coder — COMMENTED** [review]
A few comments for review. Great work so far.

File: src/components/Header.test.js (hunk 1 above)
Comment (DeonFranklin) [reply src/components/Header.test.js]: Fixed this it statement.

File: src/pages/Home.js (hunk 2 above)
Comment (DeonFranklin) [reply src/pages/Home.js]: Done

File: src/pages/NotFound.test.js (hunk 4 above)
Comment (DeonFranklin) [reply src/pages/NotFound.test.js]: Fixed

File: src/pages/Home.test.js (hunk 3 above)
Comment (DeonFranklin) [reply src/pages/Home.test.js:9]: Gracias

File: src/App.test.js [hunk 5]
```diff
@@ -1,8 +1,33 @@
...
+    expect(renderedHeader.length).toEqual(1)
+    expect(renderedFooter.length).toEqual(1)
+  })
+  it("providing a route '/' to the home component",() => {
+    //Arrange
+    const renderedApp = shallow(<App/>)
+    //Act
+    const renderedHomeRoute = renderedApp.find('[path="/"]')
+   
+    console.log("rendered Home Debug", renderedHomeRoute.debug());
```
Comment (sjproctor) [inline src/App.test.js]: You can remove the console logs once you have confirmed your tests are passing.

File: src/components/Header.js [hunk 6]
```diff
@@ -1,11 +1,70 @@
...
+    this.setState({
+      isOpen: !this.state.isOpen,
+    });
+  }
+  render() {
+    return (
+      <div className="light-blue">
+        <Navbar color="light" light expand="md">
+          <NavbarBrand href="/"><h1>Cat Tinder</h1></NavbarBrand>
+          {/* <img src={catLogo} alt="" height="50px"/> */}
```
Comment (sjproctor) [inline src/components/Header.js]: You might want to consider removing this line as well as your import.

File: src/mockCats.js [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
-  {
-    id: 3,
-    name: "Toast",
-    age: 1,
-    enjoys: "getting all the attention",
-    image: "https://images.unsplash.com/photo-1592194996308-7b43878e84a6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
-  }
-]
-export default cats
+// let cats = [
```
Comment (sjproctor) [inline src/mockCats.js]: I don't think these should be commented out.

File: src/pages/CatIndex.js [hunk 8]
```diff
@@ -1,9 +1,25 @@
...
+    return(
+      <Card body key={cat.id}>
+        <CardTitle>
+          <h4>{cat.name}</h4>
+        </CardTitle>
+      </Card>
+    )
+  })}
+</Col>
+</>
```
Comment (sjproctor) [inline src/pages/CatIndex.js]: Keep your indentation clean!

**Review by sjproctor — DISMISSED** [review]
Few things to clean up.

**Review by WalkerTexas-coder — APPROVED** [review]
Good work!

**Pull request merged**